v2/bindings/csharp/SnapPluginLibCs/SnapPluginLib/CBridge.cs

```diff
@@ -1,5 +1,4 @@
 ﻿using System;
```

Comment:
Files are missing copyright headers in general

Reply:
done